src/Concerns/ComposesMessages.php

```diff
@@ -148,4 +149,20 @@ public function unpinAllMessages(): Telegraph
 
         return $telegraph;
     }
+
+    public function forwardMessage(TelegraphChat|int $fromChatId, int $messageId): Telegraph
+    {
+        $fromChatId = is_int($fromChatId) ? $fromChatId : $fromChatId->chat_id;
```

Comment:
meglio usare `$fromChat` qui

```suggestion
    public function forwardMessage(TelegraphChat|int $fromChat, int $messageId): Telegraph
    {
        $fromChatId = is_int($fromChat) ? $fromChat : $fromChat->chat_id;
```